technology/protocols.md

```diff
@@ -568,4 +568,4 @@ The server can specify formulas to use for pricing. Each formula is an expressio
 
 **Client wants to swap WETH into `1000` USDT.** Client looks up baseToken USDT and quoteToken WETH and uses the `ask` levels above. `1000` is multiplied by `0.00055` for a total of `0.55` WETH.
 
-**Client wants to swap USDT into `1` WETH.** Client looks up baseToken WETH and quoteToken USDT and uses the `ask` levels above. `1` is multiplied by `2001` for a total of `2001` WETH.
\ No newline at end of file
+**Client wants to swap USDT into `1` WETH.** Client looks up baseToken WETH and quoteToken USDT and uses the `ask` levels above. `1` is multiplied by `2001` for a total of `2001` WETH.
```

Comment:
What's different here?

Reply:
I fixed my VScode's prettier settings, and I think it fixed some type of formatting. I don't see a difference, and I'm actually confused by this